schemas/webhooks/webhook_resend.yaml

```diff
@@ -18,6 +20,7 @@ properties:
           description: ID of the resource whose webhooks are to be retriggered.
         event:
           type: string
-          enum:
```

Comment:
I cant highlight the correct line, but `resource_type` was removed in https://github.com/onfido/onfido-openapi-spec/pull/51

Reply:
Nevermind, you merged master into the branch afterwards